modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/processor/IllegalComponentException.java

```diff
@@ -7,20 +7,20 @@
 package org.mule.module.extension.internal.runtime.processor;
 
 /**
- * A {@link RuntimeException} which marks that a selected operation
+ * A {@link RuntimeException} which marks that a selected component
  * is not valid or misconfigured
  *
  * @since 4.0
  */
-public class IllegalOperationException extends RuntimeException
+public class IllegalComponentException extends RuntimeException
```

Comment:
I would rather have IllegalOperaitonException extend IllegalComponentException, but it's still good to have qualified errors for invalid source and invalid operations. That makes error handling and testing easier.